src/plugins/imagery/components/ImageControls.vue

```diff
@@ -26,27 +26,27 @@
         <div class="c-button-set c-button-set--strip-h">
             <button class="c-button t-btn-zoom-out icon-minus"
                     title="Zoom out"
-                    @click="handleZoomButton(-1)"
+                    @click="incrementZoomFactor(-1)"
```

Comment:
how about,
1. rename these to `zoomIn()` and `zoomOut()` 
2. create a new `const ZOOM_FACTOR = 1;` so later we can modify this factor if needed, (or `zoomFactor` from props? )
3. create new function `processZoom(zoomFactor)` 
4. zoomIn can call this with `ZOOM_FACTOR` and zoomOut can call this with `-ZOOM_FACTOR`

Reply:
hmm what about use `zoomfactor` instead of 1/-1?